nav_msgs/Path path
float64 average_velocity
float64 startTime
---
nav_msgs/Path path_timestamped